### ISSUE TITLE
Can't continue training the CustomDetectionModel on colab

### ISSUE THREAD

**Issue by kingfo-cq** [OlafenwaMoses/ImageAI#701]
Hi all:
    When i was trying to continue training my CustomDetectionModel on colab i got the error output below,and it couldn't continue training and save new models.
Generating anchor boxes for training images and annotation...
Average IOU for 9 anchors: 0.83
Anchor Boxes generated.
Detection configuration saved in  /content/drive/MyDrive/CTSR-imageAI/data set/json/detection_config.json
Evaluating over 2190 samples taken from /content/drive/MyDrive/CTSR-imageAI/data set/validation
Training over 5258 samples  given at /content/drive/MyDrive/CTSR-imageAI/data set/train
Training on: 	['Alert to cross wind', 'Attention to children', 'BEWARE PEDESTRIAN', 'Intersection', 'Lift the no overtaking', 'Meet and give way', 'Narrow bridge', 'Narrowing on both sides', 'Narrowing on the left', 'No Left or Right Turn', 'No Overtaking', 'No Tooting', 'No U-turn', 'No entry', 'No left turn', 'No long-term parking', 'No parking', 'No right turn', 'No straight', 'No straight and left turns', 'No straight and right turns', 'Pay attention to livestock', 'Release the speed limit of 40km', 'Reverse curve', 'Right side narrowing', 'STOP FOR CHECK', 'Sharp left turn', 'Sharp right turn', 'Slow down and give way', 'Speed limit 100km', 'Speed limit 10km', 'Speed limit 120km', 'Speed limit 15km', 'Speed limit 20km', 'Speed limit 30km', 'Speed limit 40km', 'Speed limit 50km', 'Speed limit 5km', 'Speed limit 60km', 'Speed limit 70km', 'Speed limit 80km', 'T-junction-left', 'T-junction-right', 'T-junction-straight', 'Traffic Lights Ahead', 'roundabout', 'steep slope-down', 'steep slope-up', 'stop at intersection', 'winding road']
Training with Batch Size:  6
Number of Training Samples:  5258
Number of Validation Samples:  2190
Number of Experiments:  200
Training with transfer learning from pretrained Model
WARNING:tensorflow:`period` argument is deprecated. Please use `save_freq` to specify the frequency in number of batches seen.
WARNING:tensorflow:`epsilon` argument is deprecated and will be removed, use `min_delta` instead.
WARNING:tensorflow:Model failed to serialize as JSON. Ignoring... Layer YoloLayer has arguments in `__init__` and therefore must override `get_config`.
/usr/local/lib/python3.7/dist-packages/tensorflow/python/keras/engine/training.py:1844: UserWarning: `Model.fit_generator` is deprecated and will be removed in a future version. Please use `Model.fit`, which supports generators.
  warnings.warn('`Model.fit_generator` is deprecated and '
/usr/local/lib/python3.7/dist-packages/tensorflow/python/data/ops/dataset_ops.py:3504: UserWarning: Even though the tf.config.experimental_run_functions_eagerly option is set, this option does not apply to tf.data functions. tf.data functions are still traced and executed as graphs.
  "Even though the tf.config.experimental_run_functions_eagerly "
Epoch 1/200
1216/7016 [====>.........................] - ETA: 55:07 - loss: nan - yolo_layer_3_loss: nan - yolo_layer_4_loss: nan - yolo_layer_5_loss: nan

here's my code:
from imageai.Detection.Custom import DetectionModelTrainer
import os

os.chdir('/content/drive/MyDrive/CTSR-imageAI')
cur_path = os.getcwd()

data_dir = cur_path+'/'+'data set/'
model_dir = cur_path+'/'+'data set/models/detection_model-ex-014--loss-0012.515.h5'

trainer = DetectionModelTrainer()
trainer.setModelTypeAsYOLOv3()
trainer.setDataDirectory(data_directory=data_dir)
trainer.setTrainConfig(object_names_array=["Alert to cross wind","Attention to children","BEWARE PEDESTRIAN","Intersection","Lift the no overtaking","Meet and give way","Narrow bridge","Narrowing on both sides","Narrowing on the left","No entry","No Left or Right Turn","No left turn","No long-term parking","No Overtaking","No parking","No right turn","No straight and left turns","No straight and right turns","No straight","No Tooting","No U-turn","Pay attention to livestock","Release the speed limit of 40km","Reverse curve","Right side narrowing","roundabout","Sharp left turn","Sharp right turn","Slow down and give way","Speed limit 100km","Speed limit 10km","Speed limit 120km","Speed limit 15km","Speed limit 20km","Speed limit 30km","Speed limit 40km","Speed limit 50km","Speed limit 5km","Speed limit 60km","Speed limit 70km","Speed limit 80km","steep slope-down","steep slope-up","stop at intersection","STOP FOR CHECK","T-junction-left","T-junction-right","T-junction-straight","Traffic Lights Ahead","winding road"], batch_size=6, num_experiments=200, train_from_pretrained_model=model_dir)
trainer.trainModel()

the model named 'detection_model-ex-014--loss-0012.515.h5' is the latest saved model when i was using colab trained before colab auto interrupt after 24h.

Could anybody help me please?

**Comment by toolhater** [comment]
Thats alot of data and alot of classes. Have you tried running on a faster GPU? Google collab plus gives you options to use faster GPUs but warning it's not unlimited.
